packages/strapi-plugin-content-type-builder/admin/src/containers/App/actions.js

```diff
@@ -292,7 +292,7 @@ export function getDataSucceeded({ allModels, models }, connections, { data }) {
 
 export function onChangeExistingContentTypeMainInfos({ target }) {
   const value =
-    target.name === 'name'
+    target.name.split('.')[1] === 'name'
```

Comment:
is it possible that target.name doesn't contain a `.` ? In the case it is possible this would break with out of range access on an array

Reply:
In this case no it's not possible.